### DISCUSSION TITLE
My department in my University doesn't give emails to undergraduates, can't sign up.

### DISCUSSION THREAD

**Discussion by nikosszzz** [community/community#140597] [GitHub Education]
### Hi everyone,

Hello,

I've been trying to be accepted into GitHub Education, but when I use my University's name from the dropdown list (National and Kapodistrian University of Athens), it says I need a academic email from my university to be linked to my GitHub.

The department that I'm enrolled in my university doesn't provide an email address to its students, specifically undergraduates which I am one. How else can I enroll into GitHub Education? Can I somehow just use my ID card when I get it in my hands physically?

**Comment by Ahsankhalid618** [comment]
@nikosszzz There is another way to get GitHub Education benefits. Scan your valid University ID card with your full name written on it and the date of your Enrollment/Batch. Submit your documents and send them via email that your institution does not provide any email address to the students, they will verify through other documents. I hope it helps.
